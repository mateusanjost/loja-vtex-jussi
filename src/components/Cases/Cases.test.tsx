import TestRenderer from "react-test-renderer";
import CasesComponent from "./Cases";

describe("Cases", () => {
  it("should match snapshot", () => {
    const testRenderer = TestRenderer.create(<CasesComponent />);
    const jsonComponent = testRenderer.toJSON();

    expect(jsonComponent).toMatchSnapshot();
  });
});
