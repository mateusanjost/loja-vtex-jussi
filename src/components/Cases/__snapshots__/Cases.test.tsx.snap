// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cases should match snapshot 1`] = `
<div
  className="sc-hKFxyN sc-eCApnc okUlG jnHPsH"
>
  <div
    className="sc-iCoGMd dnyLUa"
  >
    <b>
      / /
    </b>
    NOSSAS SOLUÇÕES
  </div>
  <div
    className="sc-jSFjdj kloxlG"
  >
    <div
      className="sc-gKAaRy fSPsFX"
    >
      <div
        className="sc-kEqXSa gmCBTZ"
      >
        <b>
          P1
        </b>
      </div>
      <div
        className="sc-hKFxyN sc-pNWdM okUlG ggjZcS"
      >
        Nome do Produto #1
      </div>
      <div
        className="sc-hKFxyN sc-jrsJWt okUlG fAKluN"
      >
        Descrição do produto #1
      </div>
      <div
        className="sc-hKFxyN sc-iqAclL okUlG"
      >
        Feature 1
        Feature 2
        Feature 3
      </div>
      <button
        className="sc-fujyAs jjbAlx"
      >
        Ver solução
      </button>
    </div>
    <div
      className="sc-gKAaRy fSPsFX"
    >
      <div
        className="sc-kEqXSa gmCBTZ"
      >
        <b>
          P2
        </b>
      </div>
      <div
        className="sc-hKFxyN sc-pNWdM okUlG ggjZcS"
      >
        Nome do Produto #2
      </div>
      <div
        className="sc-hKFxyN sc-jrsJWt okUlG fAKluN"
      >
        Descrição do produto #2
      </div>
      <div
        className="sc-hKFxyN sc-iqAclL okUlG"
      >
        Feature 1
        Feature 2
        Feature 3
      </div>
      <button
        className="sc-fujyAs jjbAlx"
      >
        Ver solução
      </button>
    </div>
    <div
      className="sc-gKAaRy fSPsFX"
    >
      <div
        className="sc-kEqXSa gmCBTZ"
      >
        <b>
          P3
        </b>
      </div>
      <div
        className="sc-hKFxyN sc-pNWdM okUlG ggjZcS"
      >
        Nome do Produto #3
      </div>
      <div
        className="sc-hKFxyN sc-jrsJWt okUlG fAKluN"
      >
        Descrição do produto #3
      </div>
      <div
        className="sc-hKFxyN sc-iqAclL okUlG"
      >
        Feature 1
        Feature 2
        Feature 3
      </div>
      <button
        className="sc-fujyAs jjbAlx"
      >
        Ver solução
      </button>
    </div>
    <div
      className="sc-gKAaRy fSPsFX"
    >
      <div
        className="sc-kEqXSa gmCBTZ"
      >
        <b>
          P4
        </b>
      </div>
      <div
        className="sc-hKFxyN sc-pNWdM okUlG ggjZcS"
      >
        Nome do Produto #4
      </div>
      <div
        className="sc-hKFxyN sc-jrsJWt okUlG fAKluN"
      >
        Descrição do produto #4
      </div>
      <div
        className="sc-hKFxyN sc-iqAclL okUlG"
      >
        Feature 1
        Feature 2
        Feature 3
      </div>
      <button
        className="sc-fujyAs jjbAlx"
      >
        Ver solução
      </button>
    </div>
  </div>
</div>
`;
